docs/userGuide/siteJsonFile.md

```diff
@@ -268,3 +269,9 @@ To disable this validation **entirely**, you may add the following to `site.json
   ```
 
 </div>
+
+#### **`versions`**
```

Comment:
Need to update the [live preview page](https://deploy-preview-1870--markbind-master.netlify.app/userguide/glossary#live-preview) to mention that `versions` is not yet supported. This can be a future improvement :)

Reply:
Sorry, could you clarify what you mean by this! Thank you!

Reply:
We currently have live preview (hot reload) for changes made to the attributes in `site.json` except for `baseUrl`. I'm assuming the `versions` attribute does not support hot reload yet from what I can see (if I'm already serving a site using `markbind serve`, adding/removing a new version to `site.json` will not add/remove this to the site unless I restart the preview)? 

If that is the case, we will have to document this exception here
![image](https://user-images.githubusercontent.com/47470981/166265916-8f700910-dbb6-40a7-bb48-1bb300d58b35.png)

Don't have to implement this feature in this PR as it can be a future improvement :)